src/retry/client.rs

```diff
@@ -175,14 +192,41 @@ impl<D> RequestBuilder<D>
             .chain_err(|| "failed to parse uri")?;
         let mut req = Request::new(Method::GET, url);
         req.headers_mut().extend(self.headers.clone().into_iter());
-        req.headers_mut().append(header::CONTENT_TYPE, self.d.content_type());
```

Comment:
I guess this was supposed to be a `header::ACCEPT` originally, but I'm also fine if we prefer to just drop it for now.